source/tutorials/php.rst

```diff
@@ -0,0 +1,128 @@
+.. _php:
+
+Installing and Configuring PHP
+******************************
```

Comment:
To align with other tutorials, suggest changing (but note comment below): 
PHP
###

Reply:
@mvincerx   Title is very short, but it aligns with the other tutorial titles - is this OK? Maybe "PHP configuration" is better (aligns with Proxy configuration)?

Reply:
latest commit changes to "PHP and PHP-FPM"